script.js

```diff
@@ -12,10 +13,21 @@ const searchFilteredEpisodes = (input) => {
     epi.summary.toLowerCase().includes(input.toLowerCase())
 );
 }
+function getAllEpisodesLive(){
```

Comment:
Good job with this function, your fetch works well. Try adding error handling incase of any issues.